src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigurePrinter, AreaCalculator {
+    private String color;

```

Comment:
would be more idiomatic to create a constructor for color, area, instead of using setters in constructors we would call super()

Reply:
added constructor for color, but can't figure out how to make this with area, because I sould obtained it first and then add to parameters